test/movies.js

```diff
@@ -9,66 +9,72 @@ test('no movies available for a date far in the future', function (assert) {
   })
 
   api.getMovies(function (err) {
-    var futureDate = moment().add(200, 'days').format('MMM D')
+    var futureDate = moment().add(200-1, 'days').format('MMM D')
     assert.equal(err, 'No showtimes were found on ' + futureDate + '.Please select a different date.')
     assert.end()
   })
 })
 
-test('get theaters from zipcode', function (assert) {
-  api = new Showtimes(90504)
-  api.getTheaters(function (err, theaters) {
-    assert.equal(err, null)
-    assert.ok(theaters.length > 1)
-    assert.end()
-  })
-})
-
 test('get theaters from zipcode and get movie for first movie id', function (assert) {
   api = new Showtimes(90504)
   api.getTheaters(function (err, theaters) {
     assert.equal(err, null)
     api.getMovie((theaters[0].movies[0].id), function (err2, movie) {
       assert.equal(err2, null)
-      assert.ok(movie.theaters[0].showtimes.length > 0)
+      assert.ok(movie.theaters[0].showtimes.length > 0, 'movie found by id')
+      assert.end()
     })
-
-    assert.end()
   })
 })
 
-test('get theaters from lat/long', function (assert) {
+test('get movies from lat/long', function (assert) {
   api = new Showtimes('33.8358,-118.3406')
-  api.getTheaters(function (err, theaters) {
+  api.getMovies(function (err, movies) {
     assert.equal(err, null)
-    assert.ok(theaters.length > 1)
+    assert.ok(movies.length > 1, 'more than one movie')
     assert.end()
   })
 })
 
 test('get movies from lat/long', function (assert) {
-  api = new Showtimes('33.8358,-118.3406')
+  api = new Showtimes('45.531531531531535,-122.61220863200342')
   api.getMovies(function (err, movies) {
     assert.equal(err, null)
-    assert.ok(movies.length > 1)
+    assert.ok(movies.length > 1, 'more than one movie')
     assert.end()
   })
 })
 
-test('get theaters from lat/long', function (assert) {
+test('get movies from lat/long and get filtered movies for first movie name', function (assert) {
   api = new Showtimes('45.531531531531535,-122.61220863200342')
-  api.getTheaters(function (err, theaters) {
+  api.getMovies(function (err, movies) {
     assert.equal(err, null)
-    assert.ok(theaters.length > 1)
-    assert.end()
+    api.getMovies(function (err2, movies2) {
```

Comment:
No, it should not. It's ok, what I want to test here is that the result when filtering include the first movie :)